content/vault/global/partials/important-changes/breaking-changes/cve-2025-6000.mdx

```diff
@@ -5,15 +5,22 @@
 | Breaking     | 1.20.1+, 1.19.7+, 1.18.12+, 1.16.23+ | All
 
 File audit devices require explicit configuration for prefixing and cannot use
-executable file permissions.  You must set `allow_audit_log_prefixing` to `true`
-in your server configuration to enable `file` audit devices with the `prefix`
-option. Additionally, `file` audit devices cannot use file modes with executable
-permissions (e.g., 0777, 0755).
+executable file permissions.
+
+On upgrade, if you have configured a single audit device, and it is a file audit
+device with the executable [mode](/vault/docs/commands/audit/enable#command-arguments) set,
+**Vault will not unseal**. Vault `file`
+audit devices cannot use file modes with executable permissions
+(e.g., 0777, 0755), and should likely be configured with 0644 or similar.
+
+Additionally, to enable `file` audit devices with the `prefix` option, you must
+set `allow_audit_log_prefixing` to `true` in your server configuration on each
+node in your cluster.
```

Comment:
```suggestion
Vault will not unseal on upgrade if your only configured audit device is a
`file` device with the executable
[mode](/vault/docs/commands/audit/enable#command-arguments) set.

Vault `file` audit devices cannot use file modes with executable permissions
(e.g., 0777, 0755), and should be configured with `0644` permissions
(or similar).

Additionally, to enable `file` audit devices with the `prefix` option, you must
set `allow_audit_log_prefixing` to `true` in your server configuration on each
node in your cluster.
```
Style correction: write in active voice, put the most important information first